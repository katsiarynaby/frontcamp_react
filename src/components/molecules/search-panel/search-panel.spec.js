import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import SearchPanel from './index';

describe('SearchPanel', () => {
    it("SearchPanel should render correctly", () => {
        const wrapper = shallow(<SearchPanel />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
