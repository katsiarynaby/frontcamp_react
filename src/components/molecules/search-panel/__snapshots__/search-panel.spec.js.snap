// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPanel SearchPanel should render correctly 1`] = `
<div
  className="search-panel"
>
  <Logo
    className="search-panel__logo"
  />
  <h2
    className="search-panel__title"
  >
    FIND YOUR MOVIE
  </h2>
  <div
    className="search-panel__search"
  >
    <SearchField
      getValue={[Function]}
      placeholder="SEARCH"
      value=""
    />
    <SearchButton
      active={true}
      onClick={[Function]}
      value="SEARCH"
    />
  </div>
  <div
    className="search-panel__first-filter"
  >
    <SearchFilter
      activeFilter="title"
      filters={
        Array [
          Object {
            "key": "title",
            "name": "title",
          },
          Object {
            "key": "genres",
            "name": "genres",
          },
        ]
      }
      getActiveFilter={[Function]}
      title="SEARCH BY"
    />
  </div>
  <div
    className="search-panel__second-filter"
  >
    <SearchFilter
      activeFilter="release_date"
      filters={
        Array [
          Object {
            "key": "release_date",
            "name": "release date",
          },
          Object {
            "key": "vote_average",
            "name": "rating",
          },
        ]
      }
      getActiveFilter={[Function]}
      title="SORT BY"
    />
  </div>
</div>
`;
